### DISCUSSION TITLE
zh/guide/drivers/mopan

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#267] [Announcements]
# zh/guide/drivers/mopan

云盘官网链接：https://mopan.sc.189.cn/mopan/#/downloadPc 没有网页端，只有Android,`IOS,PC-Win64位,iPad,TV; :warning: AList 3.19.0版本及以上版本才能使用本驱动; ==四川电信魔盘的安全问题有点多，不建议存隐私文件。密码也不要使用常用的密码，服务器会明文存储==...

https://alist.nn.ci/zh/guide/drivers/mopan.html

**Comment by malou8** [comment]
挂载后提示

failed get objs: failed to list objs: Code:0, Message:参数错误

**Reply by baili04** [reply]
+1

**Reply by yiranxiamo** [reply]
根目录不填，直接留空就好了。

**Reply by jianghaodn** [reply]
我也是

**Reply by Wang1Hai** [reply]
兄弟们，同样的问题，解决没有

**Reply by yiranxiamo** [reply]
解决了，升级到最新版就可以挂载了。

**Comment by twister3344** [comment]
code 100030，请使用验证码登陆

**Reply by xiaoxin3242** [reply]
状态: 

Code:100030, Message:请采用验证码方式登陆

**Reply by yiranxiamo** [reply]
我也是，提示这个：Failed init storage: Code:100030, Message:请采用验证码方式登陆

**Reply by Wang1Hai** [reply]
兄弟们同样的问题解决没有

**Reply by yiranxiamo** [reply]
我是账号+密码挂载的。我以前挂载过。升级后还是一样的用密码挂载。不知道你们为啥用到验证码了

**Comment by hjs523** [comment]
failed get objs: failed to list objs: Code:100031, Message:当前版本较低，请升级最新版本，否则无法使用。

提示这个了

**Comment by doutaro** [comment]
上传文件的时候失败，提示：All attempts fail: #1: upload err,code=411 #2: upload err,code=411 #3: upload err,code=411

**Reply by Wang1Hai** [reply]
您是怎么挂载上的，这边提示使用验证码登录

**Reply by wbxyue** [reply]
兄弟，你先去注册一下魔盘的账号，设置一下密码，然后清除魔盘的数据或者直接卸载重新下，然后登录的时候选择验证码登录，然后你把接收到的验证码填到这里面就就可以挂载了

**Reply by longalong** [reply]
Failed init storage: Code:20240501, Message:设备参数异常. 。。。。。。。。这怎么破？

**Reply by longalong** [reply]
> 兄弟，你先去注册一下魔盘的账号，设置一下密码，然后清除魔盘的数据或者直接卸载重新下，然后登录的时候选择验证码登录，然后你把接收到的验证码填到这里面就就可以挂载了

Failed init storage: Code:20240501, Message:设备参数异常. 。。。。。。。。这怎么破？

**Comment by Wang1Hai** [comment]
各位大佬，请使用验证码登录，怎么处理

**Comment by yiranxiamo** [comment]
账号密码就能挂载啊




z
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alist-org/docs"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年4月5日(星期五) 中午11:46
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alist-org/docs] zh/guide/drivers/mopan (Discussion #267)





 
兄弟们，同样的问题，解决没有

**Comment by Wang1Hai** [comment]
各位大佬设备参数异常，怎么处理，之前绑定过一次，刷机后再次绑定就出现这样的问题了

**Comment by twister3344** [comment]
“ Code: 50021, Message: 参数转换异常 ” 是啥问题？

**Reply by Wang1Hai** [reply]
兄弟，解决没有？

**Comment by longalong** [comment]
Failed init storage: Code:20240501, Message:设备参数异常

**Reply by 0-smanthus** [reply]
+1,好兄弟你解决了吗

**Comment by longalong** [comment]
Failed init storage: Code:20240501, Message:设备参数异常

**Comment by leejunlon** [comment]
最近302跳转的链接下载很慢，但是在客户端中正常。
